### PR TITLE
[Mobile-7370] Release notes for android agent v6.4.0

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - release notes for android agent v6.4.0

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

**Comment by rhetoric101** [comment]
@ywang-nr, it looks like this is a draft PR that is getting reviewed by your peers, right? Could you ping our hero in the documentation channel when you're ready for a writing review?

**Comment by ywang-nr** [comment]
@rhetoric101 yes, sure thing. I will comment here once it's ready for PROD. Thank you for checking.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx
Comment (barbnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx:11]: "Andorid" s/be Android. Also recommend adding a period at end of sentence like previous bullet.

**Comment by barbnewrelic** [comment]
@ywang-nr  I noticed that your PR is waiting for a review from @cthomas-newrelic . No rush, I just wanted you to be aware that your PR also did not build correctly. We will keep your PR in draft state until you're ready to proceed. Or, let us know if you prefer for us to simply close this PR. Thanks for confirming! No rush.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx:14]: Let's update the oldest supported to `5.24.0`

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx:11]: Typo: `caused`.

Perhaps rephrase as "_Fixed crash caused by NoSuchMethodError that occurred on Android 5 and 6._"

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Review by cthomas-newrelic — DISMISSED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx
Comment (paperclypse) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx]: Looks like this got fixed

**Review by paperclypse — DISMISSED** [review]
Looks good

**Comment by paperclypse** [comment]
Hold for release until Jan 12, 2022

**Review by paperclypse — APPROVED** [review]
Looks good. Thanks @barbnewrelic!

**Comment by paperclypse** [comment]
Now that the afternoon merge is done, this will go out with tomorrow morning's merge to main.

**Pull request merged**
